examples/index.ts

```diff
@@ -563,6 +574,31 @@ async function callGetOrderbook(provider: BaseProvider) {
     console.info(req)
 }
 
+async function callGetMarketDepth(provider: BaseProvider) {
+    try {
```

Comment:
don't do try/catchs. Exceptions throw are good things, since they will fail the build

Reply:
Done